### PR TITLE
Clean up the code and add more tests in BoxOfficeSpec

### PR DESCRIPTION
**Pull request by WillemJiang**
Added more tests in the BoxOfficeSpec and clean up the code of chapter-up-and-running.

### CODE REVIEW

File: chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package com.goticks
 
-import akka.actor.{ Actor, ActorRef, Props, ActorSystem }
-
-import akka.testkit.{ TestKit, ImplicitSender, DefaultTimeout }
-
-import org.scalatest.{ WordSpecLike, MustMatchers }
+import akka.actor.{ActorRef, ActorSystem, Props}
+import akka.testkit.{DefaultTimeout, ImplicitSender, TestKit}
```
Comment (RayRoestenburg) [inline chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala]: Please format imports as:
```import akka.testkit.{ DefaultTimeout, ImplicitSender, TestKit }```
(the space after `{` and before `}`)

File: chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+import com.goticks.BoxOffice._
+import com.goticks.TicketSeller._
+import org.scalatest.{MustMatchers, WordSpecLike}
```
Comment (RayRoestenburg) [inline chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala]: Same here.

File: chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala [hunk 3]
```diff
@@ -48,5 +49,36 @@ class BoxOfficeSpec extends TestKit(ActorSystem("testBoxOffice"))
...
+      val boxOffice = system.actorOf(BoxOffice.props)
+      val noneExitEventName = "noExitEvent"
+      boxOffice ! BoxOffice.GetEvent(noneExitEventName)
+      expectMsg(None)
+
+      boxOffice ! CancelEvent(noneExitEventName)
+      expectMsg(None)
+    }
+
+    "Cancel Ticket which event is not created " in {
```
Comment (RayRoestenburg) [inline chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala]: Please change to "Cancel a ticket for an event that is not created yet"

File: chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala [hunk 4]
```diff
@@ -48,5 +49,36 @@ class BoxOfficeSpec extends TestKit(ActorSystem("testBoxOffice"))
...
+      val boxOffice = system.actorOf(BoxOffice.props)
+      val noneExitEventName = "noExitEvent"
+
+      boxOffice ! CancelEvent(noneExitEventName)
+      expectMsg(None)
+    }
+
+    "Cancel Ticket which event is created" in {
```
Comment (RayRoestenburg) [inline chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala]: Please change to "Cancel a ticket"

File: chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala (hunk 1 above)
Comment (WillemJiang) [inline chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala]: Sure. I will submit a quick fix for it.

File: chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala (hunk 3 above)
Comment (WillemJiang) [inline chapter-up-and-running/src/test/scala/com/goticks/BoxOfficeSpec.scala]: OK

**Comment by RayRoestenburg** [comment]
Thanks @WillemJiang!

**Pull request merged**